Create GRUNT.md

chucklorenz opened the pull request:
fixes #1140

oliverfoster commented:
+1

moloko commented:
+1

hrajotia commented:
great doc @chucklorenz, just one more thing, can we document few other command line options too?

e.g. (menu, theme, port)
```
grunt build --menu=adapt-contrib-boxMenu --theme=adapt-contrib-vanilla
grunt server --port=8080
```

Thanks

chucklorenz commented:
Yes @hrajotia , happy to include those options. Happy to include others I may have overlooked --just let me know.

moloko commented:
+1

lc-thomasberger commented:
+1

there is also the option to add a host, so that you can access the course from your network. defaults to localhost.
`grunt server --host=0.0.0.0`

hrajotia commented on GRUNT.md:
```diff
@@ -7,11 +7,16 @@ All Adapt grunt commands are run from the root directory of the Adapt course whe
 **build**  
-`grunt build [--languages=xx,yy]`  
-Compiles all files in `src` to create a production-ready minified/compressed version of your course. Includes a variety of error-checking tasks. Add the "languages" flag to limit which language/s should be built, e.g, `grunt build --languages="de,fr"`.  
+`grunt build [--languages=xx,yy] [--menu=xx] [--theme=xx] [--sourcedir=xx] [--outdir=xx]`  
```
small typo here.. should be `--outputdir`

hrajotia commented on GRUNT.md:
```diff
@@ -7,11 +7,16 @@ All Adapt grunt commands are run from the root directory of the Adapt course whe
 
 **dev**  
-`grunt dev`  
+`grunt dev [--languages=xx,yy] [--menu=xx] [--theme=xx] [--sourcedir=xx] [--outdir=xx]`   
```
small typo here.. should be `--outputdir`

taylortom commented:
Can we also make sure that any changes are also made to https://github.com/adaptlearning/adapt_framework/wiki/Compiling%2C-testing-and-deploying-your-Adapt-course (not suggesting this is a task for you @chucklorenz)

chucklorenz commented:
I added the two new translate tasks (knowing the tasks have not yet been released) to [Compiling, testing and deploying your Adapt course](https://github.com/adaptlearning/adapt_framework/wiki/Compiling,-testing-and-deploying-your-Adapt-course) (and cleaned up some typos, grammars, and formatting).  Others that are missing? The page could benefit from the addition of command options.

hrajotia commented:
+1 - awesome.

oliverfoster commented:
+1
